ONSdigital/eq-author-app
# Added script to automatically build and deploy design system

### What is the context of this PR?

To automate building the design system, I have added a Github Actions script which runs whenever code is pushed to the master branch. It will recompile the design system (based on the code that will be in master) and push it to the `gh-pages` branch, which will then be deployed to Github pages.

### How to review 

Have a look at the script and see if I've made some mistakes. Unfortunately, the only way to test it is to put it in Master. (I have some Storybook changes to do after this PR, so I can test if this works then.)

* Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
    * ensure it can be opened in Author;
    * then, ensure it can be viewed in Runner by pressing the **view survey** button
* Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Jira ticket for this task into the next stage of the process
